NeedleFoundation.podspec

```
@@ -11,5 +11,5 @@ Pod::Spec.new do |s|
   s.source           = { :git => 'https://github.com/uber/needle.git', :tag => "v" + s.version.to_s }
   s.source_files     = 'Sources/**/*.swift'
   s.ios.deployment_target = '9.0'
-  s.swift_version    = '4.2'
+  s.swift_versions = ['5.2', '5.3', '5.4', '5.5', '5.6']
```

Let's preserve the existing formatting

```suggestion
  s.swift_versions   = ['5.2', '5.3', '5.4', '5.5', '5.6']
```

---

Thanks, I reflected the commit suggestion!